google_forms_to_jira/autokitteh.yaml

```diff
@@ -10,12 +16,12 @@ version: v1
 project:
   name: google_forms_to_jira
   vars: 
-    - name: FORM_ID
-      value: <Google Form ID> # TODO: Replace with Google Form ID
+    - name: GOOGLE_FORM_ID
+      value: 
     - name: JIRA_PROJECT_KEY
-      value: <Jira Project Key> # TODO: Replace with Jira Project Key
+      value: 
     - name: POLL_INTERVAL
-      value: 10
+      value: 
```

Comment:
Similar to the Copilot sample - `POLL_INTERVAL` is fine as a required var with a description at the top of the file, but a reasonable default is still useful in this specific case.

Reply:
Done